container/libcontainer/handler.go

```diff
@@ -43,6 +43,8 @@ var (
 	whitelistedUlimits      = [...]string{"max_open_files"}
 	referencedResetInterval = flag.Uint64("referenced_reset_interval", 0,
 		"Reset interval for referenced bytes (container_referenced_bytes metric), number of measurement cycles after which referenced bytes are cleared, if set to 0 referenced bytes are never cleared (default: 0)")
+	disableSocketCountStats = flag.Bool("disable_socket_count_stats", false,
```

Comment:
I think this is currently covered under the "process" stats category in --disable_metrics.  Does that not work for your usecase?

Reply:
Sorry for the late reply !

No, because we want to have process stats, we just don't need the socket counts ( partly because it's too costly to collect )